susemanager-branding-oss/susemanager-branding-oss.spec

```diff
@@ -27,6 +27,7 @@ Source0:        %{name}-%{version}.tar.gz
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 BuildArch:      noarch
 Provides:       susemanager-branding = %{version}
+BuildRequires:  SUSE-Manager-Server-release
```

Comment:
Not sure I am understanding this. Do we want to avoid building the package on Uyuni?

Reply:
This package contains the license files. For Uyuni we do not have license files. So we do not need this package.
Interestingly the ExcludeArch works with noarch packages.